pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -30,70 +27,16 @@ func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineIns
 }
 
 func (l *LibvirtDomainManager) prepareMigrationTarget(vmi *v1.VirtualMachineInstance, useEmulation bool) error {
+	domain := &api.Domain{}
```

Comment:
nit: Can you define it near the usage now?

Reply:
sure